plugins/nf-amazon/src/main/nextflow/cloud/aws/batch/AwsBatchFileCopyStrategy.groovy

```diff
@@ -154,7 +154,7 @@ class AwsBatchFileCopyStrategy extends SimpleFileCopyStrategy {
     }
 
     static String uploadCmd( String source, Path target ) {
-        "nxf_s3_upload '$source' s3:/$target"
+        "nxf_s3_upload \"$source\" s3:/$target"
```

Comment:
These `uploadCmd` methods must be updated because of how they are called in `SimpleFileCopyStrategy`:
```groovy
        return """\
            IFS=\$'\\n'
            for name in \$(eval "ls -1d ${escape.join(' ')}" | sort | uniq); do
                ${stageOutCommand('$name', targetDir, mode)} || true
            done
            unset IFS""".stripIndent(true)
```